src/main/schemas/facilities.schema.json

```diff
@@ -7,6 +7,25 @@
     "code" : {
         "type": "string",
         "pattern": "^[A-Z0-9]{2,3}$"
+    },
+    "url": {
+      "type": "string"
```

Comment:
Can we constrain this using `format: "uri"`?

Reply:
I tried that, but it doesn't seem to work correctly, and any value seems to pass validation. I didn't want to validation via regex if at all possible, too many variables. Ideas?

Reply:
I got the URI format working on the schema. Also swapped the regex on email for format email, which seems to work pretty well.

Reply:
This seems to work great:

```
"url": {
      "type": "string",
      "format": "uri",
      "pattern": "^http(s)?"
 },
```

Reply:
Just tested, agreed. Will add in the pattern as well.